dev-python/axolotl-curve25519/axolotl-curve25519-0.4.1.ebuild

```diff
@@ -0,0 +1,28 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+MY_PN="python-${PN}"
+MY_PV="${PV}-2"
```

Comment:
Include that -2 into PV, and transform it back (e.g. `0.4.1_p2` would be an easy way to do it).

Reply:
Thanks, changed.